test/functional/test_framework/wallet.py

```diff
@@ -333,6 +333,19 @@ def bulk_transaction(self, tx, target_weight):
             "tx": tx_heavy,
         }
 
+    def send_self_transfer_chain(self, *, from_node=None, chain_length, utxo_to_spend=None):
```

Comment:
1c9243b40db5746575ed55d09774872f0a98b24a: `from_node` should be a required arg, like in the other `send*` methods.